system-design/app-interoperability/observability.md

```diff
@@ -1,37 +1,40 @@
+- Monitoring the container platform's health and current state. This includes aspects such as memory, CPU, and disk usage as well as cluster, node, pod, and container availability, run state, and configured resource limits. This enables end users to make decisions such as whether or not a device can support more applications, or has too many deployed.
 - Monitoring the workload orchestration agent and containerized application's state to ensure it is running correctly, performing as expected and not consuming more resource than expected.
 - To assist with debugging/diagnostics for applications encountering unexpected conditions impacting their ability to run as expected.
 
 Margo's application observability is NOT intended to be used to monitor anything outside the device such as production processes, machinery, controllers, or sensors and should NOT be used for this purpose.
 
 ## Observability Framework
 
-Instead of defining our own observability framework we have adopted the [OpenTelemetry specification](https://opentelemetry.io/docs/). OpenTelemetry is a popular open source specification that provides a common way for observability data to be used. There are several reasons why OpenTelemetry was chosen which includes:
+Application observability data is made available using [OpenTelemetry](https://opentelemetry.io/docs/). OpenTelemetry, a popular open source specification, defines a common way for observability data to be generated and consumed.
+
+There are several reasons why OpenTelemetry was chosen:
 
 - OpenTelemetry is based on a open source specification and not only an implementation
 - OpenTelemetry is widely adopted
 - OpenTelemetry has a large, and active, open source community
 - OpenTelemetry provides [SDKs](https://opentelemetry.io/docs/languages/) for many popular languages if people wish to use them
-- The OpenTelemetry community project has a lot of existing tooling that is useful such as telemetry [receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver) for Kubernetes, Docker and the host system.
-- OpenTelemetry is vendor agnostic and there are a lot of popular open source, and for purchase, backend solutions available for consuming and displaying observability data on the market.
+- The OpenTelemetry community project has reusable components such as telemetry [receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver) for Kubernetes, Docker and the host system making integration easier.
+- OpenTelemetry is vendor agnostic.
+- There are a lot of open-source and paid products available for consuming and displaying observability data.
```

Comment:
I'd cut this bullet point. Not sure it adds value.

Reply:
updated